themes/bootstrap3/js/record.js

```diff
@@ -312,6 +312,12 @@ function removeCheckRouteParam() {
 
 function recordDocReady() {
   removeCheckRouteParam();
+  $('.record-tabs .nav-tabs li').attr('aria-selected', 'false');
+  $('.record-tabs .nav-tabs .initiallyActive').attr('aria-selected', 'true');
+  $('.record-tabs .nav-tabs a').on('shown.bs.tab', function shownTab(e) {
+    $('.record-tabs .nav-tabs li').attr('aria-selected', 'false');
+    $(e.target).parent().attr('aria-selected', 'true');
+  });
```

Comment:
This code is great! A comment or two might make it more readable.

Reply:
Thanks for the review, @crhallberg. I'll wait to see if @aidaluuppala decides to add comments before merging this.